virt/virt-4-12-release-notes.adoc

```diff
@@ -88,6 +88,7 @@ The SVVP Certification applies to:
 
 
 //CNV-20181 Comply with cluster-wide crypto policy
+* The {VirtProductName} Operator now reads the cluster-wide xref:../security/tls-security-profiles.adoc#tls-profiles-kubernetes-configuring_tls-security-profiles[TLS security profile]  and propagates it to the {VirtProductName} components, including virtualization, storage, networking, and infrastructure.
```

Comment:
Should this mention from where it is being read?

Reply:
That's a good idea; would that be the `APIServer` custom resource?